### ISSUE TITLE
Auth Management UI also?

### ISSUE THREAD

**Issue by giorgio79** [sam0829/firebase_auth_ui#5]
Love the idea of this plugin.

Could we have a UI for managing the auth? Eg signout and deletion of user from app?

**Comment by sam0829** [comment]
Hi, this plugin is wrapper of native [firebase ui](https://firebase.google.com/docs/auth/android/firebaseui) library. I don't think it provides UI for such operations but I'll check in details. Meanwhile you can use [logout](https://github.com/sam0829/firebase_auth_ui#logout) method for manually logging out. Currently there's no method in plugin for deleting user which I'll add in upcoming updates :)

**Comment by sam0829** [comment]
Hi @giorgio79, I've added a method to delete a user for now. I am not sure how much it'll be helpful to have a UI for managing auth as for most of the operations, it's a matter of calling a single method.

Of course, if you are talking about functionalities like change password then it makes sense to have a UI. I am still not sure if native Android/iOS Firebase UI has something for this. I've closed the issue for now and if I find the way, I'll re-open this :)
